client/login/wp-login/login-buttons.tsx

```diff
@@ -75,13 +80,13 @@ const LoginButtons = ( {
 
 		return (
 			<Button
-				className="social-buttons__button button"
+				className={ classNames( 'social-buttons__button button', { disabled: true } ) }
```

Comment:
This shouldn't be `{ disabled: isDisabled }`?

Reply:
Yes totally, it escaped my debug phase!